src/styles/dungeon.less

```
@@ -65,6 +65,14 @@
   -webkit-filter: brightness(0%); /* Safari 6.0 - 9.0 */
   filter: brightness(0%);
   opacity: 0.2;
+  body.cyborg &,
+  body.darkly &,
+  body.slate &,
+  body.superhero & {
+    -webkit-filter: brightness(100); /* Safari 6.0 - 9.0 */
+    filter: brightness(100);
+    opacity: 0.5;
```

Reduce the opacity slightly:
```suggestion
    opacity: 0.3;
```